wiki/Skinning/Skinner/en.md

```diff
@@ -0,0 +1,7 @@
+---
+stub: true
+---
+
+# Skinner
```

Comment:
same as storyboarder, idk. I can't see these being useful

Reply:
would be useful as links and expanding on notable skinners in the future and their impact on the game e.g. LuigiHann. Same can be said for storyboarders

Reply:
still not too sure if we really need them, but i left a comment that they can be expanded on by someone who has more knowledge in those areas